kamu-cli/src/app.rs

```diff
@@ -219,6 +219,16 @@ fn load_config(workspace_layout: &WorkspaceLayout, catalog: &mut CatalogBuilder)
             .flink
             .clone()
             .unwrap(),
+        datafusion_image: config
+            .engine
+            .as_ref()
+            .unwrap()
+            .images
+            .as_ref()
+            .unwrap()
+            .datafusion
+            .clone()
+            .unwrap(),
```

Comment:
Yeah... we really should improve our config service to avoid this crazy chain. Need to do a survey of config crates to see if there's something suitable.